src/PartialResponse.Core/Field.cs

```diff
@@ -16,15 +17,20 @@ public struct Field
         /// <summary>
         /// Initializes a new instance of the <see cref="Field"/> structure.
         /// </summary>
-        /// <param name="value">The value of the field.</param>
-        public Field(string value)
+        /// <param name="parts">The value of the field.</param>
+        public Field(params string[] parts)
```

Comment:
The reason was to move responsibility of splitting field parts to parser to not to make a dependency on the delimiter options. But looking at it now it made me realize that it will probably have to have it and Wildcard should be also part of the delimiter options.